src/common/dcsvg.cpp

```diff
@@ -224,50 +270,79 @@ wxString wxGetBrushStyleName(wxBrush& brush)
     return brushStyle;
 }
 
-wxString wxGetBrushFill(wxBrush& brush)
+wxString wxGetBrushFill(wxBrush const& brush)
 {
     wxString s;
     wxString brushStyle = wxGetBrushStyleName(brush);
 
     if (!brushStyle.IsEmpty())
-        s = wxS(" fill=\"url(#") + brushStyle + brush.GetColour().GetAsString(wxC2S_HTML_SYNTAX).substr(1) + wxS(")\"");
+        s = wxS("fill=\"url(#") + brushStyle + Col2SVG(brush.GetColour()).substr(1) + wxS(")\"");
 
     return s;
 }
 
-wxString wxCreateBrushFill(wxBrush& brush)
+wxString wxRenderMode(wxSVGShapeRenderingMode const& style)
+{
+    wxString mode;
+    switch (style) {
+        case wxSVG_SHAPE_RENDERING_OPTIMIZE_SPEED:
+            mode = wxS("optimizeSpeed");
+            break;
+        case wxSVG_SHAPE_RENDERING_CRISP_EDGES:
+            mode = wxS("crispEdges");
+            break;
+        case wxSVG_SHAPE_RENDERING_GEOMETRIC_PRECISION:
+            mode = wxS("geometricPrecision");
+            break;
+        case wxSVG_SHAPE_RENDERING_AUTO:
+        default:
```

Comment:
Would be better to avoid `default` inside the switch to let the compiler warn us if a new element is added but not handled here.